packages/jaeger-ui/src/model/ddg/types.tsx

```diff
@@ -27,10 +27,10 @@ export enum EViewModifier {
 }
 
 export enum EDdgDensity {
-  MostConcise = 'MC',
-  UpstreamVsDownstream = 'UvD',
-  PreventPathEntanglement = 'PPE',
-  ExternalVsInternal = 'EvI',
+  ExternalVsInternal = 'ext-vs-int',
```

Comment:
not sure if we want to keep these as short as possible due to url length constraints faced elsewhere.
Additionally, while not affected by these changes but these changes remind me of it, does navigating from one DDG to another preserve URL params such as density? 🤔 
TODO for self: confirm density/showOp is passed to navigated ddg

Reply:
layout settings are not passed along, will update